Maskbook/Scene/App/FIleService/View/FileServiceSelectFileSourceView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  FileServiceSelectFileTypeView.swift
+//  Maskbook
+//
+//  Created by xiaojian sun on 2022/6/8.
+//  Copyright © 2022 dimension. All rights reserved.
+//
+
+import SwiftUI
+
+struct FileServiceSelectFileSourceView: View {
+    let viewModel: FileServiceSelectFileSourceViewModel
+
+    var body: some View {
+        VStack {
+            Spacer().frame(height: 64)
+            ForEach(FileServiceSelectFileSourceViewModel.LocalFileSourceItem.allCases, id: \.self) { item in
```

Comment:
Maybe this is better `typealias Item = FileServiceSelectFileSourceViewModel.LocalFileSourceItem`?